### ISSUE TITLE
How are conflicts resolved?

### ISSUE THREAD

**Issue by aphex3k** [getumbrel/umbrel-community-app-store#8]
I'm debating setting up a community app store but I want to host apps that are the same as in the original umbrel app store but with different configurations/modifications.

e.g.

I made a configuration change to the bitcoind app and would upload that to my community app store. How does umbrel resolve the issue that multiple stores are providing apps by the same identifier?

**Comment by jakubkarafa** [comment]
Yes, you should be fine as long as your app has a different `id:` in `umbrel-app.yml`. This is usually enforced by using the format:

```
yourstoreid-app-name
```

For example:

* Official app: `bitcoind`
* Your version: `mystore-bitcoind`

That said, I **wouldn’t recommend installing both the official app and your modified version at the same time**, even if the IDs are different. They could:

* Conflict on ports
* Interfere with shared data volumes
* Cause confusion in the Umbrel UI

**Best practice:** give your app a unique ID and avoid parallel installs of conflicting apps.
